include/umf/providers/provider_os_memory.h

```diff
@@ -103,10 +105,11 @@ umfOsMemoryProviderParamsDefault(void) {
     umf_os_memory_provider_params_t params = {
         UMF_PROTECTION_READ | UMF_PROTECTION_WRITE, /* protection */
         UMF_MEM_MAP_PRIVATE,                        /* visibility mode */
-        NULL,                                       /* numa_list */
-        0,                                          /* numa_list_len */
-        UMF_NUMA_MODE_DEFAULT,                      /* numa_mode */
-        0                                           /* part_size */
+        NULL,                  /* name of a shared memory file */
```

Comment:
please add an info that this is an optional parameter

Reply:
Done